### ISSUE TITLE
Proxysql on Debian should now run under the proxysql user account

### ISSUE THREAD

**Issue by tullis** [voxpupuli/puppet-proxysql#43]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 4.10
- Ruby: 2.3.3
- Distribution: Debian
- Module version: 1.1.2

## How to reproduce (e.g Puppet code you use)

Configure the Percona repository
Apply the proxysql class
Install version 1.4.6 of proxysql

## What are you seeing

The configuration files are changed from `proxysql` user and group ownership to `root`.

The service fails to start with permissions errors.

## What behaviour did you expect instead

Proxysql service starting

## Output log

```

ar 28 09:37:08 staging-test1 proxysql[28009]: Starting ProxySQL: 2018-03-28 09:37:08 main.cpp:317:ProxySQL_Main_process_global_variables(): [WARNING] Unable to open config file /etc/proxysql.cnf
Mar 28 09:37:08 staging-test1 proxysql[28009]: Could not chdir into datadir: /var/lib/proxysql . Error: Permission denied
```

## Any additional information you'd like to impart

It looks like the user and group that proxysql runs under on Debian has changed since this module was last updated.

**Comment by ekohl** [comment]
This should be a nice patch to start contributing. You probably need to add a version check at https://github.com/voxpupuli/puppet-proxysql/blob/9f10815793ad5827fa0112022bcd2302928b2027/manifests/params.pp#L27-L32

Could you come up with a patch?

**Comment by tullis** [comment]
I'll see what I can do.

**Comment by tullis** [comment]
Pull Request here https://github.com/voxpupuli/puppet-proxysql/pull/44
I could not find a version which ever used `root` so I simply merged the case statements for Ubuntu and Debian.

**Comment by tullis** [comment]
This is broadly addressed by the updates that are set out in #47
